Sources/Orbit/Support/Accessibility/AccessibilityID.swift

```diff
@@ -1,57 +1,73 @@
+    static let emptyStateTitle              = Self(rawValue: "orbit.emptystate.title")
+    static let emptyStateDescription        = Self(rawValue: "orbit.emptystate.description")
+    static let emptyStateButton             = Self(rawValue: "orbit.emptystate.button")
+    static let fieldLabel                   = Self(rawValue: "orbit.field.label")
+    static let fieldMessage                 = Self(rawValue: "orbit.field.message")
+    static let fieldMessageIcon             = Self(rawValue: "orbit.field.message.icon")
+    static let inputPrefix                  = Self(rawValue: "orbit.input.prefix")
+    static let inputSuffix                  = Self(rawValue: "orbit.input.suffix")
+    static let inputValue                   = Self(rawValue: "orbit.input.value")
+    static let inputFieldPasswordToggle     = Self(rawValue: "orbit.inputfield.password.toggle")
+    static let keyValueKey                  = Self(rawValue: "orbit.keyvalue.key")
+    static let keyValueValue                = Self(rawValue: "orbit.keyvalue.value")
+    static let listChoiceTitle              = Self(rawValue: "orbit.listchoice.title")
+    static let listChoiceIcon               = Self(rawValue: "orbit.listchoice.icon")
+    static let listChoiceDescription        = Self(rawValue: "orbit.listchoice.description")
+    static let listChoiceValue              = Self(rawValue: "orbit.listchoice.value")
+    static let passwordStrengthIndicator    = Self(rawValue: "orbit.passwordstrengthindicator")
+    static let radioTitle                   = Self(rawValue: "orbit.radio.title")
+    static let radioDescription             = Self(rawValue: "orbit.radio.description")
+    static let tileTitle                    = Self(rawValue: "orbit.tile.title")
+    static let tileIcon                     = Self(rawValue: "orbit.tile.icon")
+    static let tileDescription              = Self(rawValue: "orbit.tile.description")
+    static let tileDisclosureButtonLink     = Self(rawValue: "orbit.tile.disclosure.buttonlink")
+    static let tileDisclosureIcon           = Self(rawValue: "orbit.tile.disclosure.icon")
+
+    public let rawValue: String
+
+    public init(rawValue: String) {
+        self.rawValue = rawValue
+    }
 }
 
-extension View {
+public extension View {
+
+    /// Uses the specified identifier to identify the view inside a component.
+    @available(iOS 14.0, *)
+    func accessibilityIdentifier(_ accessibilityID: AccessibilityID) -> some View {
+        self.accessibilityIdentifier(accessibilityID.rawValue)
+    }
 
-    /// Uses the specified Orbit identifier to identify the view inside a component.
-    func accessibility(_ identifier: AccessibilityID) -> some View {
-        self.accessibility(identifier: identifier.rawValue)
+    /// Uses the specified identifier to identify the view inside a component.
+    @available(iOS, introduced: 13.0, deprecated: 100000.0, renamed: "accessibilityIdentifier(_:)")
```

Comment:
Shouldn't that be `deprecated: 14.0`? 🤔

Reply:
I copied the SwiftUI declaration, but it is true that we now specify it exactly. According to docs, it should be 16.1, right?

https://developer.apple.com/documentation/swiftui/presentedwindowcontent/accessibility(identifier:)